src/Utils/ExifCleaner.php

```diff
@@ -0,0 +1,102 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Utils;
+
+use Psr\Log\LoggerInterface;
+use Symfony\Component\DependencyInjection\ParameterBag\ContainerBagInterface;
+use Symfony\Component\Process\Exception\ProcessFailedException;
+use Symfony\Component\Process\ExecutableFinder;
+use Symfony\Component\Process\Process;
+
+class ExifCleaner
+{
+    protected const EXIFTOOL_COMMAND_NAME = 'exiftool';
+    protected const EXIFTOOL_ARGS_COMMON = [
+        '-overwrite_original', '-ignoreminorerrors',
+    ];
+    protected const EXIFTOOL_ARGS_SANITIZE = [
+        '-GPS*=', '-*Serial*=',
+    ];
+    protected const EXIFTOOL_ARGS_SCRUB = [
+        '-all=',
+        '-tagsfromfile', '@',
+        '-colorspacetags', '-commonifd0', '-orientation', '-icc_profile',
+        '-XMP-dc:all', '-XMP-iptcCore:all', '-XMP-iptcExt:all',
+        '-IPTC:all',
+    ];
+    protected const EXIFTOOL_TIMEOUT_SECONDS = 10;
+
+    private readonly ?string $exiftoolPath;
+    private readonly ?string $exiftool;
+    private readonly int $timeout;
+
+    public function __construct(
+        private readonly ContainerBagInterface $params,
+        private readonly LoggerInterface $logger,
+    ) {
+        $this->exiftoolPath = $params->get('exif_exiftool_path');
+        $this->timeout = $params->get('exif_exiftool_timeout') ?? self::EXIFTOOL_TIMEOUT_SECONDS;
```

Comment:
getting a 500 with no `.env` changes:

```
Cannot assign string to property App\Utils\ExifCleaner::$timeout of type int
```

Reply:
noted. changed to use `%env(int:default::EXIF_EXIFTOOL_TIMEOUT)%`, think I'll need to run it for some time to make sure that it actually fixes it

Reply:
well fwiw I pulled it in with no changes on my end and it fixed the 500 for me